Update etcd-operator to latest release

lachie83 opened the pull request:
bump etcd and etcd-operator version and update notes

lachie83 committed:
bump etcd and etcd-operator version and update notes

lachie83 commented:
cc @chancez

chancez commented on stable/etcd-operator/templates/NOTES.txt:
```diff
@@ -8,12 +8,12 @@ Not enabling cluster, the ThirdPartResource must be installed before you can cre
   kubectl get pods -l etcd_cluster={{ .Values.cluster.name }} --namespace {{ .Release.Namespace }} -w
 2. Confirm etcd cluster is healthy
-  $ kubectl run --rm -i --tty --env="ETCDCTL_API=3" --env="ETCDCTL_ENDPOINTS=http://{{ .Values.cluster.name }}-0000:2379" etcd-test --image quay.io/coreos/etcd --restart=Never -- /bin/sh -c 'watch -n1 "etcdctl  member list"'
+  $ kubectl run --rm -i --tty --env="ETCDCTL_API=3" --env="ETCDCTL_ENDPOINTS=http://{{ .Values.cluster.name }}-client:2379" etcd-test --image quay.io/coreos/etcd --restart=Never -- /bin/sh -c 'watch -n1 "etcdctl  member list"'
```
I didn't think etcd-operator deployed a service by default. Are you sure this is correct?

lachie83 replied:
Looks like a change in behavior. This never used to be the case as you mentioned. Deploy this update and take a look.

rimusz replied:
LGTM

lachie83 commented:
@chancez have you had a chance to run this up?

chancez commented:
@lachie83 I haven't, I'm fairly busy atm and the cluster I've been using for TPR stuff is having some troubles, but overall the changes look reasonable if the service' name is `${clustername}-client`.

lachie83 committed:
Update to latest operator
Add ability to create rbac roles and bindings for clusters with rbac enabled
Update helpers to provide 63 limit names

lachie83 commented:
@rimusz @chancez push another commit to change this to the latest (now 0.3.2) and also include conditional creation of rbac roles/rolebindings for clusters with rbac enabled. I've tested this on a 1.6 cluster with rbac enabled.

rimusz commented:
LGTM

lachie83 committed:
update documentation

lachie83 committed:
change rbac default to off

lachie83 commented:
@prydonius @rimusz can you take another pass at this?
